app/src/main/java/com/lagradost/cloudstream3/ui/player/OfflinePlaybackHelper.kt

```diff
@@ -0,0 +1,44 @@
+package com.lagradost.cloudstream3.ui.player
+
+import android.content.ContentUris
+import android.content.Context
+import android.net.Uri
+import androidx.core.content.ContextCompat.getString
+import com.lagradost.cloudstream3.CommonActivity.activity
+import com.lagradost.cloudstream3.R
+import com.lagradost.cloudstream3.utils.ExtractorUri
+import com.lagradost.cloudstream3.utils.UIHelper.navigate
+import com.lagradost.safefile.SafeFile
+
+object OfflinePlaybackHelper {
```

Comment:
Both these use an implicit acitvity, make them pass in activity instead of context